entries/iteration.qmd

```diff
@@ -14,32 +14,39 @@ for incrementally building or improving things. The reason we don't completely u
     - While making tasks (Issues), keep them as small as is reasonable and as descriptive and targeted as possible.
     - As we discuss each task, we collectively assign team members to tasks (assign to the Issues).
-    - We'll review the number of tasks and the aim of the iteration, then discuss and agree on how long the iteration should be (can range from 2-6 weeks, but prefer shorter iterations over longer ones).
-    - After the iteration length is decided, we'll schedule an approximately 2 hour debrief/retrospective meeting on the end date.
-    - Either during or afterward the planning meeting, each team member individually estimates how much time they might need on their assigned tasks, taking into consideration how much time they have available during their work on the iteration. They'll use the the Iteration's Roadmap Board to track and estimate their time spent on tasks.
+        - Assign only one person per Issue so that the Project board can be kept organized and so that each Issue has someone who is responsible for them. Anyone can help with the Issue though.
+    - We'll review the number of tasks and the aim of the iteration, then discuss and agree on how long the iteration should be (can range from 2-6 weeks, but prefer shorter iterations over longer ones, for instance, 3 weeks).
+    - After the iteration length is decided, we'll schedule an approximately 2-3 hour debrief/retrospective as well as the next iterations planning meeting on the end date (and start of next).
+    - (Optional) Either during or afterward the planning meeting, each team member individually estimates how much time they might need on their assigned tasks, taking into consideration how much time they have available during their work on the iteration. They'll use the the Iteration's Roadmap Board to track and estimate their time spent on tasks.
+
 3. Throughout the iteration, we will:
-    - Have our [update meetings](update-meetings.qmd) to discuss progress, next steps, and any struggles or barriers (with the work or the iteration/process).
-    - Work on our assigned tasks.
+    - Have our [update meetings](update-meetings.qmd) to discuss progress, next steps, and any struggles or barriers (with the work or the iteration/process), as well as to showcase the demo.
+    - Schedule multi-person impromptu meetings as necessary and as relevant for a given task or issue.
+    - Work on our assigned tasks and coordinate with others assigned to multi-person tasks.
     - Review Pull Requests from others.
     - Add more Issues if required. If the Issue is relevant to the current iteration, then add it to the Project Board so that we can determine who will self-assign it during one of our meetings. If the Issue isn't relevant, don't add it to the Project Board, and we will save it for future iterations to work on. 
```

Comment:
```suggestion
    - Add more Issues if required. If the Issue is relevant to the current iteration, then add it to the Project Board so that we can determine who will self-assign it during one of our meetings. If the Issue isn't relevant, don't add it to the Project Board but to the relevant repository instead, and we will save it for future iterations to work on. 
```

Reply:
Seems to be a good way of handling it.